bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -175,27 +176,39 @@ num_templates = len(h5py.File(args.template_bank, "r")['template_hash'])
 tmin, tmax = parse_template_range(num_templates, args.template_fraction_range)
 logging.info('Analyzing template %s - %s' % (tmin, tmax-1))
 
-# Create dictionary for trigger files indexed on ifo name
-trigs = {}
+class MultiifoTrigs(object):
+    """store trigger info in parallel with ifo name and shift vector"""
+    def __init__(self):
+        self.ifos = []
+        self.to_shift = []
+        self.singles = []
+
+trigs = MultiifoTrigs()
 for i in range(len(args.trigger_files)):
     logging.info('Opening trigger file %s: %s' % (i,args.trigger_files[i]))
     reader = ReadByTemplate(args.trigger_files[i],
-                                               args.template_bank,
-                                               args.segment_name,
-                                               args.veto_files)
-    trigs[reader.ifo] = reader
+                            args.template_bank,
+                            args.segment_name,
+                            args.veto_files)
+    ifo = reader.ifo
+    trigs.ifos.append(ifo)
+    # time shift is subtracted from pivot ifo time
+    trigs.to_shift.append(-1 if ifo == args.pivot_ifo else 0)
```

Comment:
Ok, so if I understand, this tells which way to shift a particular trigger time. i.e. time + DIRECTION * SLIDE. Where direction can be 0 or -1 in this case. Makes sense. (might also be extendable in the future which is also good).

Reply:
yes, or +1 (for some reason the 2-ifo coinc code seems to be written to positively shift ifo1, at least that's what I deduced from the way the time difference was done in the 2-ifo pta stat)